lib/modules/onc_program_procedure.yml

```diff
@@ -340,20 +334,14 @@ procedure:
           * "Device";
           * "DiagnosticReport";
           * "DocumentReference";
-          * "Encounter";
           * "Goal";
           * "Immunization";
-          * "Location";
```

Comment:
The `Location` element seems to be present in the TLV list (though it's not in the SUT list)

Reply:
Is this an issue in this file?  I'm having trouble finding where Location is on the TLV side, but I might just be missing it.  I do know that ONC's test procedure is wrong, in that Location is on the TLV side on this, but I think I fixed it?

Reply:
I'm not seeing this issue, so I'm going to go ahead and resolve this.  The fact that the TP itself has this problem in it means that we either fix it and be inconsistent with the TP, or leave it to be consistent with the TP (but wrong).  So either way it isn't a great situation.  I've reported it to ONC so hopefully in the next version it will be addressed.